app/build.gradle

```diff
@@ -56,7 +56,7 @@ dependencies {
 
 	fullImplementation 'com.crashlytics.sdk.android:crashlytics:2.9.9'
 	implementation 'org.openobservatory.measurement_kit:android-libs:0.10.1-1-android.2'
-	implementation 'com.github.xanscale.LocalhostToolkit:app:0f80dba55e'
+	implementation 'com.github.xanscale.LocalhostToolkit:app:68279cae65'
```

Comment:
Do you mind tagging a release with a semantically significant version rather than a hash?

Reply:
done